### PR TITLE
Added introduction draft

### PR DESCRIPTION
**Pull request by ishaankolluri**
I can adjust the wording as needed. Let me know!

### CODE REVIEW

File: tex/introduction.tex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+\chapter{Introduction}
+Extend is a declarative programming language meant to support spreadsheet-like functionality. It contains features such as complex calculations on large inputs of data, side-effect free and immutable values, and automatic formula adjustments relative to rows and columns. Extend is compiled to the LLVM (Low Level Virtual Machine) intermediate representation, which in turn is reduced to machine assembly. Extend takes inspiration from software such as Microsoft Excel, which allows users to link several formulae on dependent groups of data together, but takes this technology a step further by allowing users to encapsulate such calculations as functions.
+
+\section{Inspiration \& Use Cases}
+	
+	\subsection{Inspiration}
+	Microsoft Excel and Google Sheets find issues at scale when users need to process more and more complicated calculations on several different sets of data, sometimes at scheduled intervals. Extend was conceptualized as a standalone application that removes the manual element of entering new inputs. It brings the best of spreadsheets and computation into one product.
```
Comment (Neitsch) [inline tex/introduction.tex:15]: complex instead of complicated

Comment (Neitsch) [reply tex/introduction.tex:15]: It brings the simplicity of spreadsheets while offering the expressiveness of high level languages into one product.

Something along those lines maybe?

Comment (ishaankolluri) [reply tex/introduction.tex:15]: Yeah, that sounds a little bit better. Will workshop this today after I finish working on the tutorial section.

Comment (oracleofnj) [reply tex/introduction.tex:15]: I would like to say something along these lines (wording needs work):

The design goal of the language was to be "a spreadsheet you can compile". In order to turn the spreadsheet environment into a flexible compiled programming language, there were three main things that needed to be changed about the way an interactive spreadsheet like Excel works: (1) the language needs reusable functions as opposed to copy-and-paste-a-block-of-cells, (2) cell ranges needed to be created with dynamic (runtime-determined) dimensions; and (3) in order to support more complex data structures, we decided to allow complex contents for cell values as opposed to single numbers or strings. With those changes in mind, we attempted to keep the semantics as similar as possible to traditional spreadsheet programs. This meant implementing a dynamically typed and fairly "forgiving" language.

Comment (ishaankolluri) [reply tex/introduction.tex:15]: Really like this. Will try to adapt.

**Review by Neitsch — APPROVED** [review]

**Pull request merged**